apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -57,8 +58,21 @@ function WelcomeTourCard( {
 	return (
 		<Card className="welcome-tour-card" isElevated>
 			<CardOverlayControls onDismiss={ onDismiss } onMinimize={ onMinimize } />
-			<CardMedia>
-				<img alt={ __( 'Editor Welcome Tour', 'full-site-editing' ) } src={ imgSrc } />
+			{ /* TODO: Update selector for images in @wordpress/components/src/card/styles/card-styles.js */ }
+			<CardMedia className="welcome-tour-card__media">
+				<picture>
+					{ imgSrc.mobile && (
+						<source
+							srcSet={ imgSrc.mobile.src }
+							type={ imgSrc.mobile.type }
+							media={ getMediaQueryList( '<480px' )?.media }
```

Comment:
Instead of hardcoding `'<480px'`, can we import the `MOBILE_BREAKPOINT` const from the same package? I think this should help our future selves understand a bit better the use of `@automattic/viewport` here.

Reply:
Added in https://github.com/Automattic/wp-calypso/pull/59052